README.md

```diff
@@ -25,6 +25,8 @@ The design philosophy of Habitat is to prioritize simulation speed over the brea
 Habitat-Sim is typically used with
 [Habitat-Lab](https://github.com/facebookresearch/habitat-lab), a modular high-level library for end-to-end experiments in embodied AI -- defining embodied AI tasks (e.g. navigation, instruction following, question answering), training agents (via imitation or reinforcement learning, or no learning at all as in classical SensePlanAct pipelines), and benchmarking their performance on the defined tasks using standard metrics.
 
+## [Questions or Comments? Join the AI Habitat community discussions forum.](https://github.com/facebookresearch/habitat-lab/discussions)
```

Comment:
This is good, but need to remove link to Slack at L240 as well.

Reply:
Thanks, didn't realize this link existed. :laughing: 
Took the opportunity to update that section.